packages/bus-rdf-update-quads/lib/ActorRdfUpdateQuadsDestination.ts

```diff
@@ -52,7 +52,6 @@ export abstract class ActorRdfUpdateQuadsDestination extends ActorRdfUpdateQuads
     action: IActionRdfUpdateQuads,
   ): Promise<IActorRdfUpdateQuadsOutput> {
     const execute = (): Promise<void> => Promise.all([
-      action.quadStreamInsert ? destination.insert(action.quadStreamInsert) : Promise.resolve(),
```

Comment:
Since the race condition can probably also occur between `quadStreamDelete` and `deleteGraphs` (and also inserts), we may just want to await each operation separately.

Something like this should work:
```
const execute = async (): Promise<void> => {
  await action.quadStreamDelete ? destination.delete(action.quadStreamDelete);
  await action.deleteGraphs ?
        destination.deleteGraphs(
          action.deleteGraphs.graphs,
          action.deleteGraphs.requireExistence,
          action.deleteGraphs.dropGraphs,
        );
  ...
}
```

Reply:
I changed it as you suggested, although I am not sure about the exact order of operations that we should perform. I went with `delete -> deleteGraphs -> createGraphs -> insert`. When we introduce `update`, we might have a problem.